src/user.ts

```diff
@@ -61,18 +66,6 @@ export async function getUserData(ctx: RequestContext<ContextData>, handle: stri
         ),
     };
 
-    const dataToStore: PersonData = {
```

Comment:
Removed this as it wasn't being used

Reply:
:D This is actually a bug - we should be passing `dataToStore` on line 76 🙈

Reply:
Updated in https://github.com/TryGhost/ActivityPub/pull/6/commits/a32d9356488ea158df674275a553982ab60ea2ff